### ISSUE TITLE
passing arrays as items prop

### ISSUE THREAD

**Issue by pagrawl3** [babotech/react-adaptive-grid#1]
Hey

Tried adding arrays to the Grid through the items prop. However, I'm getting an error
> Warning: Failed propType: Invalid prop `items` of type `Array` supplied to `AdaptiveGrid`, expected instance of `List`. Check the render method of `FeedStream`

Are only Immutable.js lists supported ?

Thanks

```javascript
 const props = {
			ItemComponent: CardWrapper,
			items: filteredFeed, //array
			minWidth: 200,
		}

return (
	<section className="feed-stream-wrapper">
	{
		<Grid {...props}/>
	}
	</section>
);
```

**Comment by galkinrost** [comment]
Yes, at now only Immutable List  is available in items. At first we had alowed using of a regular array, but there was a performance issue because we need to do fromJS on every update of an array
